alien/CMakeLists.txt

```diff
@@ -56,18 +56,33 @@ arcane_framework_set_if_unset(Arctools_ROOT ${ARCFRAMEWORK_ROOT}/arctools PATH "
 
 list(APPEND CMAKE_MODULE_PATH ${ARCFRAMEWORK_ROOT}/_common/build_all/cmake)
 
-# Add Alien only if blas and boost found
+#first try to look for MKL
+message(STATUS "Looking for mkl BLAS")
+set(BLA_VENDOR Intel10_64lp_seq)
 find_package(BLAS)
-find_package(Boost OPTIONAL_COMPONENTS program_options)
-if (BLAS_FOUND AND Boost_program_options_FOUND)
-    add_subdirectory(standalone)
-    set(ALIEN_FOUND YES)
-    set(FRAMEWORK_INSTALL YES)
-    if (WIN32) # Not yet ready...
-#        if (BUILD_ARCANE_INTERFACE)
-#            add_subdirectory(ArcaneInterface)
-#        endif ()
-    elseif (BUILD_ARCANE_INTERFACE)
-            add_subdirectory(ArcaneInterface)
-    endif ()
+if(BLAS_FOUND)
+    add_definitions(-DALIEN_USE_MKL)
 endif()
+
+if(NOT BLAS_FOUND)
+    message(STATUS "Looking for OpenBLAS BLAS")
+    set(BLA_VENDOR OpenBLAS)
+    find_package(BLAS REQUIRED)
+    set(CBLAS_INCLUDE_DIR CBLAS_INCLUDE_DIR-NOTFOUND)
+    find_path(CBLAS_INCLUDE_DIR cblas.h HINTS ENV OPENBLAS_ROOT PATH_SUFFIXES openblas include REQUIRED)
+    message(STATUS "cblas.h found in ${CBLAS_INCLUDE_DIR}")
+    include_directories(${CBLAS_INCLUDE_DIR})
+endif()
```

Comment:
What to we do for other BLAS implementations? ARMPL, BLIS, ...

Reply:
We don't use them at this time so we have to be pragmatic: solving the problems we have now.

Reply:
As long as @grospelliergilles and @stdcm are ok with that in framework. However, I will not limit to openBLAS or MKL in the outside Alien because we need support for other libraries.

Reply:
alien/CMakeLists.txt is not supposed to be inside Alien ? 
May be there is some improvement to do in the workflow.

Reply:
I agree with @cedricchevalier19 that the ultimate goal is to have a clean way to detect blas and not be too specific to an implementation.
But for the current release I think the priority is to make the build works with IFPEN configuration.